config.yaml

```diff
@@ -1,4 +1,10 @@
 options:
+  upf-mode:
+    type: string
+    default: af_packet
+    description: |
+      Either `af_packet` (default) or `dpdk`.
+      Please see README.md to learn about all required parameters.
```

Comment:
The descriptions here are accessible to users when they run `juju config`, not the readme. Instead of pointing to the README, it's preferable to place all the information needed to configure dpdk here. Ex. the fact that we need hugepages.

Reply:
I removed this reference. Now all configs required by the DPDK are described accordingly.